tsl/src/chunk_api.c

```diff
@@ -1627,3 +1627,63 @@ chunk_api_get_chunk_relstats(PG_FUNCTION_ARGS)
 {
 	return chunk_api_get_chunk_stats(fcinfo, false);
 }
+
+TS_FUNCTION_INFO_V1(ts_chunk_get_dimension_slices);
+Datum
+ts_chunk_get_dimension_slices(PG_FUNCTION_ARGS)
+{
+	Oid relid = PG_ARGISNULL(0) ? InvalidOid : PG_GETARG_OID(0);
+	const Chunk *chunk = ts_chunk_get_by_relid(relid, false);
+	Hypertable *ht;
+
+	if (NULL == chunk)
+		ereport(ERROR,
+				(errcode(ERRCODE_INVALID_PARAMETER_VALUE),
+				 errmsg("must be a hypertable or chunk")));
+
+	ht = ts_hypertable_get_by_id(chunk->fd.hypertable_id);
+	PG_RETURN_TEXT_P(cstring_to_text(chunk_api_dimension_slices_json(chunk, ht)));
```

Comment:
Why not return JSONB here as opposed to text?

Reply:
The function is removed as not needed.